doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_matter_thread.rst

```diff
@@ -186,7 +186,7 @@ To build the sample with Matter over Wi-Fi support run the following command:
 
 .. code-block:: console
 
-   west build -b nrf54h20dk_nrf54h20_cpuapp -- -DCONF_FILE=prj_no_dfu.conf -DSHIELD=nrf700x_nrf54h20dk -DCONFIG_CHIP_WIFI=y
+   west build -b nrf54h20dk_nrf54h20_cpuapp -- -DFILE_SUFFIX=no_dfu -DSHIELD=nrf700x_nrf54h20dk -DCONFIG_CHIP_WIFI=y
```

Comment:
```suggestion
   west build -b nrf54h20dk_nrf54h20_cpuapp -- -DSHIELD=nrf700x_nrf54h20dk -DCONFIG_CHIP_WIFI=y
```
Hmm currently we don't support any Matter samples on nrf54h20 in NCS, but if we can run it there won't be any suffixes needed.